citi-onboarding/celeiro-dos-bambas
# Feature/navbar

## What I did

- Creation of the NavBar section.

## How to test

1. In terminal:
   1. `git checkout main`
   1. `git pull`
   1. `git checkout feature/impactFix`
   1. `git pull`
   1. `yarn run develop` in `server/` folder
   1. `yarn run start` in `client/` folder
 2. Go to [http://localhost:3000/](http://localhost:3000/)
 3. Check if the adjustments in the NavBar sections are as expected, if the drop shadow is working and the screen changes are good.
